package/build_boards_manager_package.sh

```diff
@@ -88,6 +88,7 @@ $SED 's/recipe.hooks.core.prebuild.1.pattern.*//g' \
 ver_define=`echo $plain_ver | tr "[:lower:].\055" "[:upper:]_"`
 echo Ver define: $ver_define
 echo \#define ARDUINO_ESP8266_GIT_VER 0x`git rev-parse --short=8 HEAD 2>/dev/null` >$outdir/cores/esp8266/core_version.h
+echo \#define ARDUINO_ESP8266_GIT_DESC `git describe --tags 2>/dev/null` >>$outdir/cores/esp8266/core_version.h
```

Comment:
A few lines above (can't comment on the line directly...) there is
```
$SED 's/recipe.hooks.core.prebuild.1.pattern.*//g' \
```
Since you have added 2.pattern, please add one more line, then run the script and check the the prebuild hooks are removed from platform.txt in the built archive.

Reply:
Done. I had to rebase to push the fix.